src/build/mdn-comments.ts

```diff
@@ -115,35 +45,31 @@ function insertComment(
   }
 }
 
+function extractSlug(slug: string): string[] {
```

Comment:
Why reorder the function? Almost thought this is an entirely new function with the same name.. Keeping its position should help reducing the diff.